src/primitives/transaction.h

```diff
@@ -667,8 +668,8 @@ class CTransactionBase
     virtual bool CheckVersionIsStandard   (std::string& reason, int nHeight) const = 0;
     virtual bool CheckInputsAvailability  (CValidationState &state) const = 0;
     virtual bool CheckOutputsAvailability (CValidationState &state) const = 0;
-    virtual bool CheckSerializedSize      (CValidationState &state) const = 0;
 
+    bool CheckSerializedSize (CValidationState &state) const;
```

Comment:
If we want a single `CheckSerializedSize` there must be a single max size for both txs and certs. Remove `MAX_CERT_SIZE ` in consensus.h

Reply:
Done